bin/define_env.dart

```diff
@@ -60,6 +60,12 @@ void main(List<String> argv) {
   final envSettings = loadEnvSettings(opts['settings'] as String);
   isEnvValid(dotEnv.env, envSettings);
 
+  // If the path to the env library is defined it creates and writes the library
+  if (envSettings.fields.isNotEmpty) {
```

Comment:
Execute this if provided as an option.
You can probably do this as `--generate` and `--no-generate` options, with the default being `generate`